### PR TITLE
Display field description in field list

### PR DESCRIPTION
**Pull request by rthouvenin**
Addresses  #6316

### CODE REVIEW

**Review by mazameli — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Adding the description in though is going to require some rebalancing of the other elements for each list item in order to make things look right.

### Current appearance in the branch

image

### Desired

image

**Comment by rthouvenin** [comment]
Ah my bad, I tested the change on a v0.33 build I had handy, and did notice that the styling of the page changed when I rebased my commit on master.

I made some adjustment to the style, looks better? I didn't move the raw name to the right as suggested in the image, as I found the current way more legible.

One more thing I forgot to mention when I created the PR: the Edit mode doesn't let the user modify the description. Should it?

Thanks for the review!

**Comment by rthouvenin** [comment]
@mazameli Did you get a chance to review my changes

**Comment by mazameli** [comment]
@rthouvenin, sincere apologies for letting this PR slip through the cracks! I'm just going to put a couple of spacing suggestions in the code in your PR shortly and then I think we'll be able to merge this.

File: frontend/src/metabase/reference/components/Field.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import cx from "classnames";
 import pure from "recompose/pure";
 
 const Field = ({ field, foreignKeys, url, icon, isEditing, formField }) => (
-  <div className={cx(S.item)}>
+  <div className={cx(S.item, "p1", "mt1", "border-top")}>
```
Comment (mazameli) [inline frontend/src/metabase/reference/components/Field.jsx:22]:
```suggestion
  <div className={cx(S.item, "pt1", "border-top")}>
```

File: frontend/src/metabase/reference/components/Field.jsx [hunk 2]
```diff
@@ -116,6 +116,11 @@ const Field = ({ field, foreignKeys, url, icon, isEditing, formField }) => (
         </div>
         <div className={F.fieldOther} />
       </div>
+      {field.description && (
+        <div className={cx(S.itemSubtitle, { mt1: isEditing })}>
```
Comment (mazameli) [inline frontend/src/metabase/reference/components/Field.jsx:120]:
```suggestion
        <div className={cx(S.itemSubtitle, "mb2", { mt1: isEditing })}>
```

**Review by mazameli — CHANGES_REQUESTED** [review]
With these two spacing changes, we should be good to merge this.

**Comment by rthouvenin** [comment]
I rebased (locally) the  branch on 0.37.2 and compared with/without the suggestions. The rebase didn't require any manual merge and and page looks crispier with your suggestion.
Thanks for looking into it, better late than never :)

**Review by mazameli — APPROVED** [review]

**Comment by mazameli** [comment]
@camsaul or @robdaemon, I'm unsure why there are three BE tests failing on this PR, which only touches one FE file. What should I do here?

**Comment by robdaemon** [comment]
@mazameli the Oracle and Vertica failures are because this is a third-party PR - they don't have access to certain parameters, so nothing to worry about there. The Cloverage failure is a flake. Feel free to merge this as-is.

**Pull request merged**